src/swarm/neo/AddrPort.d

```diff
@@ -25,11 +25,15 @@ public struct AddrPort
     import core.sys.posix.arpa.inet: ntohl, ntohs, htonl, htons, inet_ntop;
     import core.stdc.string: strlen;
 
-    import ocean.util.container.map.model.StandardHash;
+    import swarm.Const : NodeItem;
 
+    import ocean.util.container.map.model.StandardHash;
     import ocean.core.Test;
     import swarm.util.Verify;
 
+    /// Minimum length required for an address format buffer.
+    public static const AddrBufLength = INET_ADDRSTRLEN;
```

Comment:
I would play it safe here and I would go with `INET6_ADDRSTRLEN`.

Reply:
Updated.